modules/administration_manual/pages/configuration/files/file_sharing_configuration.adoc

```diff
@@ -110,38 +101,30 @@ Check this option to enable users to re-share files shared with them.
 
 Check this option to confine sharing within group memberships.
 
-[NOTE]
-====
-This setting does not apply to the Federated Cloud sharing feature.
-If xref:configuration/files/federated_cloud_sharing_configuration.adoc[Federated Cloud Sharing] is enabled, users can still share items with any users on any instances (_including the one they are on_) via a remote share.
-====
+NOTE: This setting does not apply to the Federated Cloud sharing feature. +
+If xref:configuration/files/federated_cloud_sharing_configuration.adoc[Federated Cloud Sharing] is enabled,
+users can still share items with any users on any instances (_including the one they are on_) via a remote share.
 
 === Allow users to send mail notification for shared files
 
-
 Check this option to enable users to send an email notification to every ownCloud user that the file is shared with.
 
 === Exclude groups from sharing
 
-
 Check this option to prevent members of specific groups from creating any file shares in those groups.
 When you check this, you'll get a dropdown list of all your groups to choose from.
 Members of excluded groups can still receive shares, but not create any.
 
 === Allow username autocompletion in share dialog
 
-
 Check this option to enable auto-completion of ownCloud usernames.
 
 === Restrict enumeration to group members
 
-
 Check this option to restrict auto-completion of ownCloud usernames to only those users who are members of the same group(s) that the user is in.
 
-[NOTE]
-====
-ownCloud does not preserve the mtime (modification time) of directories, though it does update the mtimes on files. See "Wrong folder date" when syncing.
-====
+NOTE: ownCloud does not preserve the mtime (modification time) of directories, though it does update
+the mtimes on files. See "Wrong folder date" when syncing.
```

Comment:
Please link "Wrong folder date" to the relevant section of the documentation.

Reply:
There is no reference available. I searched for various text variants and for mtime with no results.
I deleted the Sentence therefore.